### DISCUSSION TITLE
Kaishi 1.5k deck template

### DISCUSSION THREAD

**Discussion by ajbjc** [yomidevs/yomitan#1367] [Q&A]
So, I have yomitan connected to Ammi, and the deck I'm using is Kaishi 1.5k. I can choose to select the Kaishi 1.5k deck and model, but I need to set it up so that it roughly matches the way the Kaishi cards are done.

The fields are:

- Word*
- Word reading*
- Word meaning*
- Word furigana*
- Word audio*
- Sentence**
- Sentence meaning
- Sentence furigana
- Sentence audio
- Notes**
- Pitch accent
- Pitch accent notes**
- Frequency**
- Picture

* These are fine, though I find the glossary, which is done with {glossary-brief} to not be brief enough
** These aren't exactly important

The rest I'm having issues with. For reading, I'm using the online version of mokuro. When I select a word in the bubble that comes up, I can add the word but it doesn't include the sentence. Meaning I'm not sure how to do since you need a translation. For sentence furigana, it doesn't do the whole sentence, just the word, and it doesn't format it at all, no furigana or anything. The value I'm using for sentence is {close-prefix}<b>{cloze-body}</b>{cloze-suffix}< I also tried {sentence} as well. Sentence audio I'm not sure how to generate. Pitch accent in the Kaishi cards was generated with AJT. And for pictures, I want to use a screenshot of where it came from.

**Comment by Kuuuube** [comment]
Closing as this is quite old and I assume no longer an issue. Feel free to re-open it. You will likely get better help for these kinds of questions in the discord server. https://discord.gg/YkQrXW6TXF
